### PR TITLE
fix: Fix TradeMarketAutoOpenChatFeature (as 2.1 changed its chat message) [skip ci]

### PR DESCRIPTION
**Pull request by kristofbolyai**

### CODE REVIEW

File: common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java [hunk 1]
```diff
@@ -21,15 +21,16 @@ public class TradeMarketAutoOpenChatFeature extends Feature {
     // Type the price in emeralds or type 'cancel' to cancel:
     // Type the amount you wish to buy or type 'cancel' to cancel:
     // Type the item name or type 'cancel' to cancel:
-    private static final Pattern TYPE_TO_CHAT_PATTERN = Pattern.compile("^§6Type the .* or type 'cancel' to cancel:$");
+    private static final Pattern TYPE_TO_CHAT_PATTERN = Pattern.compile(
```
Comment (magicus) [inline common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java:24]: What happened to "Type the price in emeralds"? Replacing `.*` with `item name` seems too narrow..?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java:24]: Oh, hm, yes, I might not have seen that.

Comment (magicus) [reply common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java:24]: Now that I have a bit more time to be able to review PRs, I'd appreciate if you'd give me some time to look at them. I understand that we have a deadline coming up real soon now, and some fixes really need to be pushed straight away, but at least giving me a few hours would be nice. :)

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/features/trademarket/TradeMarketAutoOpenChatFeature.java:24]: Basically, these were my last efforts now :) I am out for the weekend, Ill be only on my macbook to merge stuff tomorrow.

**Pull request merged**